LifeCycleAssessment_oM/EnclosuresScope.cs

```diff
@@ -0,0 +1,50 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using BH.oM.Base;
+
+namespace BH.oM.LifeCycleAssessment
+{
+    [Description("PLACEHOLDER DESCRIPTION")]
+    public class EnclosuresScope : BHoMObject
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Enclosure walls are inclusive of the opaque exterior wall assemblies of a building")]
+        public EnclosuresCurtainWalls EnclosuresCurtainWalls { get; set; } = null;
+        
+        [Description("Enclosure curtain walls are large sheets of transparent glazing on the building exterior")]
+        public EnclosuresWalls EnclosuresWalls { get; set; } = null;
+        
+        [Description("Enclosure windows are are openings in the building exterior, which consist of framing and glazing")]
+        public EnclosuresWindows EnclosuresWindows { get; set; } = null;
+        
+        [Description("Enclosure doors are are openings in the building exterior, which consist of framing and panels")]
+        public EnclosuresDoors EnclosuresDoors { get; set; } = null;
```

Comment:
I would advise we avoid setting these to null initially, perhaps better to have them instantiated as new instances of their respective object type (`new EnclosuresCurtainWalls();` etc.)